doc/api/crypto.md

```diff
@@ -2509,24 +2509,25 @@ or it might contain wildcards (e.g., `*.example.com`). Because host name
 comparisons are case-insensitive, the returned subject name might also differ
 from the given `name` in capitalization.
 
-### `x509.checkIP(ip[, options])`
+### `x509.checkIP(ip)`
 
 <!-- YAML
 added: v15.6.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/41571
+    description: The `options` argument has been removed since it had no effect.
 -->
 
 * `ip` {string}
-* `options` {Object}
-  * `subject` {string} `'always'` or `'never'`. **Default:** `'always'`.
-  * `wildcards` {boolean} **Default:** `true`.
-  * `partialWildcards` {boolean} **Default:** `true`.
-  * `multiLabelWildcards` {boolean} **Default:** `false`.
-  * `singleLabelSubdomains` {boolean} **Default:** `false`.
 * Returns: {string|undefined} Returns `ip` if the certificate matches,
   `undefined` if it does not.
 
 Checks whether the certificate matches the given IP address (IPv4 or IPv6).
 
+Only `iPAddress` subject alternative names are considered, and they must match
```

Comment:
```suggestion
Only IP address subject alternative names are considered, and they must match
```

Reply:
My intention was to avoid confusion with other subject alternative names that look like IP addresses, e.g., `http://8.8.8.8/` (stored as a `uniformResourceIdentifier`) or `8.8.8.8` (stored as a `dNSName`). But the X.509 terminology probably doesn't make it any clearer.

Reply:
Yes, I was wondering why `iPAddress` was used. It makes sense but it is not very clear. How about adding an example?